test/unit/error.c

```diff
@@ -612,32 +613,136 @@ static void
 test_undefined_error_code(void)
 {
 	header();
-	plan(4);
+	plan(6);
 
 	ok(strcmp(tnt_errcode_str(box_error_code_MAX), "ER_UNKNOWN") == 0);
 	ok(strcmp(tnt_errcode_desc(box_error_code_MAX), "Unknown error") == 0);
+	ok(tnt_errcode_record(box_error_code_MAX) == NULL);
 	ok(strcmp(tnt_errcode_str(ER_TEST_FIRST - 1), "ER_UNKNOWN") == 0);
 	ok(strcmp(tnt_errcode_desc(ER_TEST_FIRST - 1), "Unknown error") == 0);
+	ok(tnt_errcode_record(ER_TEST_FIRST - 1) == NULL);
```

Comment:
Looks like this belongs to the previous commit.

Reply:
No it's good. We only introduce `tnt_errcode_record()` in this commit.